library/ajax/weno_pharmacy_search.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * weno pharmacy search.
```

Comment:
Two things.
1. If this is weno specific, then should be in the module
2. Need an acl to control access to this script (use the same acl as is controlled to access the script that calls this ajax script)